src/frontend/src/redux/system/actions.ts

```diff
@@ -30,10 +31,24 @@ export function logOut() {
 
 export function refreshLocalAuth(inputToken: string, inputId: string) {
   // refresh the 'max-age' for cookie while user is active
-  setCookie(inputToken, inputId);
+  setLogInCookie(inputToken, inputId);
   return {
     type: LOG_IN,
     userId: inputId,
     authToken: inputToken
   };
 }
+
+export function OECILogIn(username: string, password: string): any {
+  return (dispatch: Dispatch) => {
+    return apiService(dispatch, {
+      url: '/api/oeci_login',
+      data: { oeci_username: username, oeci_password: password },
+      method: 'post',
+      authenticated: true,
+      withCredentials: true
+    }).then((response: any) => {
+      history.push('/record-search');
```

Comment:
This can be a todo for later, but instead of manually changing the history, it would be better to have routing logic that reads from the Redux state (are we authed with OECI?) and determines what page to show that way.

Reply:
I can throw a conditional statement that will only redirect if the client has the `oeci_token` in browser.

Previous to this implementation we were doing something similar, just using `react-router` `<Redirect to='/record-search' />` instead. 

https://github.com/codeforpdx/recordexpungPDX/blob/58947092cbe202835374c28529cc43a154f8edd6/src/frontend/src/components/OeciLogin/index.tsx#L65